Zend/zend_gdb.c

```diff
@@ -0,0 +1,148 @@
+/*
+   +----------------------------------------------------------------------+
+   | Zend Engine                                                          |
+   +----------------------------------------------------------------------+
+   | Copyright (c) 1998-2017 Zend Technologies Ltd. (http://www.zend.com) |
```

Comment:
```suggestion
   | Copyright (c) Zend Technologies Ltd. (http://www.zend.com)           |
```

As done in the previous commits, the year ranges in headers of the source code files can be omitted now... Thank you.